scripts/utils/bdlClassnameManager

```diff
@@ -0,0 +1,225 @@
+    return a.length > b.length ? -1 : 1;
+}
+
+function printHelp() {
+    console.log(`Usage: ${process.argv[1].split('/').pop()} operation fileName [--verbose]`);
+    console.log('\nSupported operations:');
+    console.log('\tcheck\t\tcheck to see if a file contains deprecated values (returns error if it does)');
+    console.log('\tswap\t\treplace known-deprecated variable names in a JS or SCSS file');
+}
+
+function printVerboseOperationMessage(fileName, matches) {
+    console.log(colors.cyan('File name:'), colors.white(fileName));
+    console.log(colors.cyan('Found:'), matches);
+}
+
+async function handleWriteFile(matches, fileName, fileToParse, verboseMode) {
+    try {
+        if (matches.length > 0) {
+            await writeFile(fileName, fileToParse);
+            console.log(colors.cyan(`Replaced ${matches.length} classname value${matches.length !== 1 ? 's': ''} in:\t`), fileName);
+        } else if (verboseMode) {
+            console.warn(colors.yellow('No changes needed to file:\t'), fileName);
+        }
+    } catch (error) {
+        console.error(colors.red('Cannot write updated file to disk\n'), error);
+        process.exit(1);
+    }
+}
+
+async function main() {
+    // argv 0 and 1 are the node instance and the script name respectively.
+    // argv 2 is the operation: check, swap, append, extend
+    const operation = process.argv[2];
+
+    // argv 3 is the file (and extension)
+    const fileName = process.argv[3];
+
+    if (!fileName) {
+        console.error(colors.red('Missing parameter:'), colors.white('fileName'));
+        process.exit(1);
+    }
+    if (!Object.keys(conversionMap).length) {
+        console.error(colors.red('Missing values in: '), colors.white('conversionMap'));
+        process.exit(1);
+    }
+    const fileType = fileName.split('.').pop();
+    const verboseMode = process.argv[4] === '--verbose';
+    const extendMode = operation === 'extend'
+    const currentConversionMap = extendMode ? invert(conversionMap) : conversionMap;
+
+    let reBadName;
+    if (fileType === 'scss') {
+        const converter = entry => `\\${scssPrefix}${entry}(?!-)`;
+        reBadName = new RegExp(
+            Object.keys(currentConversionMap)
+                .map(converter)
+                .sort(lengthSorter) // sort order matters here for regex
+                .join('|'), 'g'
+        );
+    } else if (fileType === 'js') {
+        // The majority of .js classnames are surrounded by quotation marks
+        // Restrict to those prefixes to avoid overrwriting variable names
+        const converter = entry => jsPrefixes.map(prefix => `${prefix}${entry}`).join('|');
+        reBadName = new RegExp(Object.keys(currentConversionMap).map(converter).join('|'), 'g');
+    } else {
+        console.error(colors.yellow('Unrecognized file type for this tool. skipping', fileName, '...'));
+        process.exit(0);
+    }
+
+    try {
+        let fileToParse = await readFile(fileName, { encoding: 'utf8' });
```

Comment:
I had to double-check that this was the file's contents rather than the file itself due to the name.